modules/tinymce/changelog.txt

```diff
@@ -25,6 +25,7 @@ Version 5.7.0 (TBD)
     Fixed incorrect return types on `editor.selection.moveToBookmark` #TINY-6504
     Fixed the type signature for `editor.selection.setCursorLocation()` incorrectly allowing a node with no `offset` #TINY-6843
     Fixed incorrect behaviour when editor is destroyed while loading stylesheets #INT-2282
+    Fixed incorrect behaviour when changing an image within a figure element causes DOM changes outside of the figure element #TINY-6592
```

Comment:
This is confusing and I'm not sure what it means/is trying to say sorry. Maybe something like this instead?
```suggestion
    Fixed figure elements incorrectly splitting from it's parent element when editing the image within #TINY-6592
```